server/service/src/sync/translations/requisition.rs

```diff
@@ -157,15 +157,18 @@ pub struct LegacyRequisitionRow {
     pub approval_status: Option<LegacyAuthorisationStatus>,
 
     #[serde(deserialize_with = "empty_str_as_option_string")]
-    #[serde(rename = "programID")]
-    pub program_id: Option<String>,
+    pub orderType: Option<String>,
+    #[serde(deserialize_with = "empty_str_as_option_string")]
+    pub periodID: Option<String>,
+    #[serde(deserialize_with = "empty_str_as_option_string")]
+    pub programID: Option<String>,
```

Comment:
what do you think of the serde_rename in this case, where field needs to be mapped twice (in pull and push) ?

Reply:
Do you mean so it's called `program_id` instead of `programID`?
I have seen this in other code, I'm fine with it either way personally. I thought the current pattern in this file wasn't re-naming so went with status quo. Happy to change though if you prefer it.

Reply:
I was pushing the pattern of rename in serde (using existing simple serde functionality as much as possible), there is no consistency on that front though.